README.md

```diff
@@ -15,20 +15,20 @@
 
 <!-- teaser-begin -->
 
-EuroCropsML is a pre-processed and ready-to-use machine learning dataset for crop type classification of agricultural parcels in Europe.
+$\texttt{EuroCropsML}$ is a pre-processed and ready-to-use machine learning dataset for crop type classification of agricultural parcels in Europe.
```

Comment:
Parts of the README.md will be visible (in addition to the readme section on Github) also in the readthedocs Documentation and package description on PyPI. All three support slightly different instruction set flavors of Markdown. So we have to be very careful about choosing formatting that works on all three platforms.

The LaTex style `texttt` currently only seems to work on Github, so we should avoid it here (same a few more times below). Is there a reason why we would prefer `texttt` over the standard markdown backticks syntax?

Some more explanations in case the answer is yes and we want to keep the LaTex:

Since we rely on the MyST parser for Markdown in our Documentation, we might just need to enable an extension for LaTex to work there, see https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#math-shortcuts

You can check a preview of the built documentation at https://eurocropsml--21.org.readthedocs.build/en/21/index.html to see if it is working correctly.

The MyST extension should fix the LaTeX issue everywhere in docs files, **except** for the README.md part, which is also used on PyPI (evething after the "teaser-begin" comment is included on PyPI). On PyPI we can not modify the parsing settings and should avoid all non-standard markdown.

Reply:
Mhh, I see. There was actually no specific reason why I changed it to `texttt`, other than I just wanted to make it more distinguishable from code snippets. I think I'd just go with the MyST extension for all files except the README, and go back to using the backticks inside the README. Or do you have another preference?

Reply:
Sounds good to me :+1:

Reply:
Should be fixed now :)